### DISCUSSION TITLE
Maintenance of www.cp2k.org/science

### DISCUSSION THREAD

**Discussion by oschuett** [cp2k/cp2k#2590] [Team Posts]
Our [science page](https://www.cp2k.org/science) has been lacking regular maintenance for the past year. I believe, this page is really important for attracting new users and funding. In order to keep it up to date someone would have to monitor the following papers for high impact citations:
 - https://dx.doi.org/10.1016/j.cpc.2004.12.014
 - https://dx.doi.org/10.1002/wcms.1159
 - https://dx.doi.org/10.1063/1.2770708

We could also include other noteworthy articles like the recent one about the [Azure cloud](https://azure.microsoft.com/en-us/blog/azure-hc-series-virtual-machines-crosses-20000-cores-for-hpc-workloads/) (thanks @hfp) and share some of it through our [Twitter channel](https://twitter.com/CP2Kproject) .

**Comment by rybkinjr** [comment]
I have updated science regularly last year, although not too often, picking
new works of general interest, i.e . high-profile journals. I believe
that's not a bad strategy: the paper should stay at the top of the list for
some time; the source should be an eye catcher. Featuring older technical
work won't be catchy.

Yours,

Vladimir



сб, 22 июн. 2019 г., 18:53 Ole Schütt <notifications@github.com>:

**Comment by oschuett** [comment]
I agree that we should only cover recent publications and not too many. One entry per month should be enough to proof that there is a steady stream of height-impact articles. Thanks for doing this!

**Comment by hfp** [comment]
If you did not like the [Azure cloud article](https://azure.microsoft.com/en-us/blog/azure-hc-series-virtual-machines-crosses-20000-cores-for-hpc-workloads/) because of being a Blog hosted on Microsoft's TecArticle site, you may enjoy the related article at [TheNextPlatform](https://www.nextplatform.com/2019/06/26/counting-the-cost-of-scaling-hpc-applications/). This article adds cost figures to the scaling study for both LiHFX and H2O-DFT-LS.

**Comment by DCM-Uni-Paderborn** [comment]
Dear Hans, 

thanks for the info, I will put that on twitter! 

Cheers, 
Thomas

**Comment by hfp** [comment]
@DCM-Uni-Paderborn btw, Microsoft was so kind to leave out the comparison with Paderborn's Noctua cluster although they wanted comparison points to ensure that the Azure nodes meet expectation. It would have beaten the Noctua results. On the other hand the Azure HC instances run higher-end CPUs (despite of crippling the core count due to running the Hypervisor on 4 cores). I think if we rerun CP2K on Noctua with the right recipe, it could perhaps show even higher performance.

**Comment by robertschade** [comment]
@hfp,@DCM-Uni-Paderborn They had the data points of Noctua in their ISC19 presentation. They violated the license (Attribution-ShareAlike 4.0 International) of the wiki because the attribution was missing as far as I can see. 
On the other hand, they have compared apples with oranges, because they ran a hybrid parallelization and optimized the number of threads per node. In contrast, the results from Noctua were pure MPI, because during the last maintenance time frame (when I performed the benchmarks) I did not have enough time to optimize the number of threads per node. This is also clearly shown at https://www.cp2k.org/performance:noctua-h2o-dft-ls.
Including threading will improve the results of Noctua. 
Hence, comparing the Azure-results to ours would have been an unfair comparison and it was only right for Microsoft to not compare to them.

**Comment by hfp** [comment]
Perhaps this tool can help:
[https://github.com/cp2k/cp2k/tree/master/tools/plan_mpi_omp](https://github.com/cp2k/cp2k/tree/master/tools/plan_mpi_omp)
which is the same tool as [here](https://github.com/hfp/xconfigure/tree/master/config/cp2k) (except with some typos fixed in documentation):
[https://github.com/hfp/xconfigure/blob/master/config/cp2k/plan.md](https://github.com/hfp/xconfigure/blob/master/config/cp2k/plan.md)
